### PR TITLE
Add ovs-cni bump script

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
Currently cnao only bumps ovs-cni manually.
Let's add a script to do it for us.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
/hold

until #449 is merged

File: hack/components/bump-ovs-cni.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+source hack/components/yaml-utils.sh
+source hack/components/git-utils.sh
+
+echo Bumping ovs-cni
```
Comment (oshoval) [inline hack/components/bump-ovs-cni.sh:8]: some times you do use '' , see this file as well , worth to keep same direction imo

File: data/ovs/001-ovs-cni.yaml [hunk 2]
```diff
@@ -59,12 +55,60 @@ spec:
                 fieldRef:
                   fieldPath: spec.nodeName
       volumes:
-        - name: localbin
```
Comment (RamLavi) [inline data/ovs/001-ovs-cni.yaml:62]: @phoracek is this necessary?

Comment (phoracek) [reply data/ovs/001-ovs-cni.yaml:62]: I think it is just a leftover that is not being used anymore

File: hack/components/bump-ovs-cni.sh [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	done
+}
+
+echo 'Bumping ovs-cni'
+OVS_URL=$(yaml-utils::get_component_url ovs-cni)
+OVS_COMMIT=$(yaml-utils::get_component_commit ovs-cni)
+OVS_REPO=$(yaml-utils::get_component_repo ${OVS_URL})
+
+TEMP_DIR=$(git-utils::create_temp_path ovs-cni)
+#trap "rm -rf ${TEMP_DIR}" EXIT
```
Comment (qinqon) [inline hack/components/bump-ovs-cni.sh:41]: missing trap ?

Comment (RamLavi) [reply hack/components/bump-ovs-cni.sh:41]: oops, Thanks!

File: hack/components/bump-ovs-cni.sh [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+	echo 'parametize manifests by object'
+	__parametize_by_object
+
+	cat <<EOF > 000-ns.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Namespace }}
+EOF
```
Comment (qinqon) [inline hack/components/bump-ovs-cni.sh:68]: Do we really need this ? namespace is already going to be there.

**Pull request merged**